### ISSUE TITLE
bug(calcite-flow): visual stuttering if reduce-motion is on

### ISSUE THREAD

**Issue by maxpatiiuk** [Esri/calcite-design-system#9455]
### Check existing issues

- [X] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates

### Actual Behavior

(see original issue in JS API: https://devtopia.esri.com/WebGIS/arcgis-js-api/issues/59317)

Slowed down video recording:

https://github.com/Esri/calcite-design-system/assets/40512816/b15eeab8-44e7-4487-8d54-dac12b6c93eb

See more video reproductions in the original issue

### Expected Behavior

There should be fewer animations and visual artifacts if reduce motion is enabled 

### Reproduction Sample

https://next.sites.afd.arcgis.com/javascript/latest/sample-code/sandbox/?sample=popup-multipleelements

### Reproduction Steps

1. [Open this sample](https://next.sites.afd.arcgis.com/javascript/latest/sample-code/sandbox/?sample=popup-multipleelements)
2. Click on any polygon on the map that has more than one feature
3. Click on the "Select feature" button in the top right - the widget disappears for a moment, before the feature list slides in - this creates needless visual noise
4. Select any feature - if user has "Reduce motion" enabled, a lot more layout shifts here

### Reproduction Version

2.8.4

### Relevant Info

See also https://github.com/Esri/calcite-design-system/issues/5072

### Priority impact

impact - p3 - not time sensitive

### Impact

Causes excessive layout shift in JS API's Popup widget if reduce motion is on

### Calcite package

- [X] @esri/calcite-components
- [ ] @esri/calcite-components-angular
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

ArcGIS Maps SDK for JavaScript

**Comment by jcfranco** [comment]
This doesn't seem to be reproducible in the latest release. @maxpatiiuk Can you confirm?

**Comment by maxpatiiuk** [comment]
Can confirm the issue is no longer reproducible in the test app provided in the original description in Calcite 3.1

https://github.com/user-attachments/assets/fa0946e6-f8fd-43bc-bda0-84d00312ae96

There is a tiny layout shift when feature-selection checkboxes appear, but is is barely noticeable. Happens on 2nd+ render too so not caused by lazy loading. Also, this might be caused on JS API side.

Slowed down version with backtracking:

https://github.com/user-attachments/assets/a2fa42de-3616-4ef1-a322-542e7ada099f

Overally, I would consider this a non issue compared to the original issue. Feel free to close as resolved.

**Comment by jcfranco** [comment]
@maxpatiiuk Thanks for testing! Closing as resolved per ☝️.

> There is a tiny layout shift when feature-selection checkboxes appear, but is is barely noticeable. Happens on 2nd+ render too so not caused by lazy loading. Also, this might be caused on JS API side.

@driskull Could you look at this and help confirm if it's a Maps SDK issue?
